aws_store/bff-service/src/index.ts

```diff
@@ -0,0 +1,77 @@
+import http from "http";
+import https from "https";
+import { URL } from "url";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const PORT = process.env.PORT || 3001;
+const SERVICES: { [key: string]: string | undefined } = {
+  product: process.env.PRODUCT_SERVICE_URL,
+  cart: process.env.CART_SERVICE_URL,
```

Comment:
with such an approach you'll need to **modify source code** and **redeploy** bff-service each time, when a new route is added.

Read again the requirements in Task 10.1:
> 3. Here's the workflow example that BFF Service should support:
> * Make requests to BFF Service with URL in the following format: `{bff-service-url}/{recipient-service-name}?var1=someValue`
> 
> * Get `recipientURL` from the env variables using `{recipient-service-name}` as a key.

the idea is to simplify maintenance. So when we want to add a new route `/new-svc`, we will just add a new environment variable like:
```
new-svc=https://new-url.com
```

and reload. no need to modify the source code!